SQL/tests/indentation/syntax_test_indentation_tsql.sql

```diff
@@ -0,0 +1,65 @@
+-- SYNTAX TEST reindent "Packages/SQL/TSQL.sublime-syntax"
+SELECT *
+FROM (
+    SELECT *
+    FROM a
+    INNER JOIN b ON a.ID = b.ID
+) alias
+WHERE 1 = 1
+GO
+
+BEGIN
+    INSERT INTO a
+    VALUES (123, 456)
+END
+
+BEGIN
+    INSERT INTO a
+    VALUES (123, 456)
+GO
+    INSERT INTO b
+    VALUES (123, 456)
+END
+
+SELECT
+a,
+b,
+c
+FROM foo
+
+SELECT a
+,b
+,c
+FROM foo
+
+CREATE TABLE hello (
+    --       @@@@@ definition
```

Comment:
Not sure why you mix symbol tests into a an indentation test file. Sure, anything is possible, but nobody will look here for those.

Reply:
I guess I got overwhelmed with how many test files there are/can be, and how much they repeat the same code being tested. Personally I don't consider symbol tests super beneficial when there are no `tmPreferences` files changing the indexing rules in the package. The scope assertions should be enough...

Reply:
Maybe symbol & indentation tests could be done at a later stage as well. The other packages are yet to have even indentation tests in them, save PHP & Java, I believe.

Reply:
Gotta start somewhere. Should I send you a PR to the PR?

Reply:
Sure, please do :) thanks